### PR TITLE
Add host roles API

### PR DESCRIPTION
**Pull request by ackintosh**
https://mackerel.io/api-docs/entry/hosts#update-roles

Add two subcommands for CLI.
`mackerel host roles update <hostId> --roleFullnames=<roleFullnames>[ <roleFullnames>...]`
`mackerel host roles delete <hostId>`

### CODE REVIEW

File: lib/mackerel/commands/host_roles.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      end
+
+      desc 'delete <hostId>', 'Delete host roles'
+      def delete(id)
+        puts JSON.pretty_generate Mackerel.update_host_roles(id, {roleFullnames: []})
+      end
+    end
+  end
+end
+
```
Comment (hsbt) [inline lib/mackerel/commands/host_roles.rb]: Can you remove this empty line?

Comment (ackintosh) [inline lib/mackerel/commands/host_roles.rb]: Removed in 5ada25e :smiley:

**Comment by hsbt** [comment]
Thank you!

**Comment by hsbt** [comment]
Released v0.3.0 included this pull request.

**Pull request merged**